2004-03-11  Theodore A. Roth  <troth@openavr.org>

	* src/disp-vcd/vcd.c:
	* src/disp-vcd/vcd.h:
	Don't inline vcd_set_clock.

Index: src/disp-vcd/vcd.c
===================================================================
RCS file: /cvsroot/simulavr/simulavr/src/disp-vcd/vcd.c,v
retrieving revision 1.3
diff -u -p -p -r1.3 vcd.c
--- src/disp-vcd/vcd.c	1 Dec 2003 05:48:35 -0000	1.3
+++ src/disp-vcd/vcd.c	11 Mar 2004 18:56:24 -0000
@@ -696,7 +696,7 @@ vcd_write_pc (int pc)
 
 /* Set the current time. */
 
-inline int
+int
 vcd_set_clock (unsigned int c)
 {
     clk = c;
Index: src/disp-vcd/vcd.h
===================================================================
RCS file: /cvsroot/simulavr/simulavr/src/disp-vcd/vcd.h,v
retrieving revision 1.2
diff -u -p -p -r1.2 vcd.h
--- src/disp-vcd/vcd.h	17 Nov 2002 00:29:20 -0000	1.2
+++ src/disp-vcd/vcd.h	11 Mar 2004 18:56:24 -0000
@@ -47,7 +47,7 @@ int vcd_trace_pc( void );
 /* Interface for disp.c */
 int vcd_write_header( void );
 
-inline int vcd_set_clock( unsigned int c );
+int vcd_set_clock( unsigned int c );
 int vcd_write_clock( void );
 
 int vcd_bind_io_reg_shortcut( char *io_reg_name, int io_reg_addr );
